tests/elliptic_curves/util.py

```diff
@@ -1,31 +1,335 @@
 import json
```

Comment:
The doctests do not work here (you can run them by calling python -m doctest -v .\tests\elliptic_curves\util.py)

Reply:
Addressed